influxdb3_write/src/lib.rs

```diff
@@ -534,7 +535,7 @@ impl ParquetFile {
 }
 
 /// The summary data for a persisted parquet file in a segment.
```

Comment:
While here, I don't think this doc comment makes sense.

Reply:
Must have been leftover from when I was writing this code the first time. Looks like a bad copy from above. Easy to remove!